### ISSUE TITLE
Clobbers Ctrl-y browser shortcut

### ISSUE THREAD

**Issue by toupeira** [passff/passff#352]
### General information. Describe your environment

- Versions
  - Operating system: Debian sid
  - Browser: Firefox 61.0.1
  - PassFF: 1.4
  - Host app: ?

Status line output: `[14:46:18] ls -> (0) no error message`

---

### Actual behaviour

PassFF uses the Ctrl-y shortcut to open its menu.

### Expected behaviour

Ctrl-y is a standard Firefox shortcut used to redo changes in text fields (counterpart to Ctrl-z / undo). A different shortcut should be used, and personally I don't need one at all so an option to disable it would be nice as well.

**Comment by toupeira** [comment]
@tuxor1337 thanks for the quick response! But my main issue is that I strongly disagree with having a default that overwrites a browser shortcut, especially one that's standard across various apps.

And yeah probably Firefox shouldn't allow overwriting such keys in the first place, but even more reason to change it in PassFF ;-)

**Comment by tuxor1337** [comment]
For me, the redo command is Ctrl+Shift+Z and that's also what it says in the menu under "edit" and on this web page: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

**Comment by toupeira** [comment]
Ah thanks, didn't know about that one! :) 

But still, Ctrl-Y is [used by Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=714832) and a [common convention](https://en.wikipedia.org/wiki/Control-Y). If you're willing to change the shortcut I'd be happy to gather some appropriate suggestions.

**Comment by tuxor1337** [comment]
@toupeira What do you think about Ctrl+Shift+L as suggested by @madduck?

**Comment by madduck** [comment]
+1 from me for `<C-S-l>`, of course, but I do wonder why this can't be made configurable? Is this also limited by the tight security woven around WebExtensions, and has to be specified in the manifest?

**Comment by tuxor1337** [comment]
Great news: It turns out that changing the browser action shortcut has already been implemented in Firefox 60+ a few months ago, see https://bugzilla.mozilla.org/show_bug.cgi?id=1421811 and https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands/update. 

The open upstream issue mentioned in my comment above (https://bugzilla.mozilla.org/show_bug.cgi?id=1215061) is about more general shortcut managing from WebExtensions which requires, of course, much more effort.

**Comment by madduck** [comment]
I just built an XPI off 020df0850b861773d6f4916c1b5c807bf92d0c59, but I cannot see an option on the preferences page. Am I missing something?

**Comment by tuxor1337** [comment]
It's the first entry in the first section of the preferences: a text input labeled as "Keyboard shortcut for toolbar menu" ("Tastenkombination für das Menü in der Werkzeugleiste", "Raccourcie clavier pour le menu de la barre d'outils", "Сочетание клавиш для меню панели инструментов"). If it doesn't appear in your build, your build seems to be based on different code. 

For testing you should not build an XPI, but use `about:debugging` as described in this guide: https://github.com/passff/passff/blob/master/docs/CONTRIBUTING.md#development

**Comment by madduck** [comment]
Yes, I saw the code, and I can confirm it works as a temporary add-on. Thanks for letting me know how to do that, I hadn't seen the update to the docs!

There is something very wonky with Firefox. If I simply install the add-on from file, then the preferences aren't updated, not even after a reboot. I cannot even install the AMO version, and then revert to the local one. The only way to get the new preference to show up was by removing the add-on and reinstalling it. I'll ask the firefox people about that.
